src/box.tsx

```diff
@@ -1,38 +1,39 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import {BoxProps} from './types/box-types'
-import {propTypes} from './enhancers'
+import { BoxProps } from './types/box-types'
+import { propTypes } from './enhancers'
 import enhanceProps from './enhance-props'
 
-export default class Box extends React.Component<BoxProps, {}> {
-  static displayName = 'Box'
+export const Box: React.FC<BoxProps> = ({ is = 'div', innerRef, children, ...props }) => {
```

Comment:
Could we test the result of the benchmark of using the class syntax vs the function syntax?

Reply:
I ran the benchmark several times as React.Component, React.SFC, React.FC and they're all pretty similar in performance. They're all around the `8,500 ops/sec` mark.

Reply:
@Rowno what do you think about this change?